### ISSUE TITLE
Bump Browserify?

### ISSUE THREAD

**Issue by ianthehenry** [jsdf/browserify-incremental#7]
I was running into some bugs around Browserify's deduplication that were fixed by [upgrading to 8.1.3](https://github.com/ianthehenry/browserify-incremental/tree/bump-browserify).

Now I see that 0.9 just came out, which I haven't tested yet.

What's the long term plan for `browserify-incremental` track `browserify` versions?

Specifying a wildcard version on `browserify` might *sort of* make sense, so that users of `browserify-incremental` have full control over the version they get (with documented "known to work" versions, perhaps). But it's also potentially risky, as it requires reading a little bit of docs. Thoughts?

**Comment by jsdf** [comment]
Yeah, I dunno, browserify breaks a lot between versions, so I'm not really sure what the best thing to do here is. If you use the API you can provide your own version of browserify but that's not helpful for the CLI use case.

**Comment by jsdf** [comment]
I've released a new version which removes the fixed browserify dependency version. Whatever version of browserify you install alongside browserify-incremental will be used. I've added a note to the 'install' section of the readme explaining this.

We'll see how it goes. If newer versions of browserify introduce more bugs, we could try going back to providing a fixed version of browserify, but also detecting versions installed along side browserify-incremental, or allowing the user to specify the location of a different version via an arg.
